index.html

```diff
@@ -154,5 +154,31 @@ <h3>About me</h3>
             </div>
         </div>
     </section>
+    <section class="contact">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
I think you might have misunderstood about using the Formspree service. You need to use your real API endpoint. To get API endpoint, first you need to sign in to the Formspree and create a new form. Inside that new form, you will see your API endpoint. I have shared some references with you. 

This is the example code: `<form action="https://formspree.io/f/mgeqgkdd" method="post" id="form">`

![Screenshot (202)](https://github.com/Newt27/Portfolio-setup_and_mobile_first/assets/111948881/5caa1d3e-2ee8-4a89-ad1c-8d17e739d4d3)